src/components/product-icon/index.tsx

```diff
@@ -9,7 +9,7 @@ import { IconVaultColor16 } from '@hashicorp/flight-icons/svg-react/vault-color-
 import { IconWaypointColor16 } from '@hashicorp/flight-icons/svg-react/waypoint-color-16'
 import { ProductIconProps } from './types'
 
-const productNamesToIcons = {
+const productSlugsToIcons = {
```

Comment:
I meant to put this refactor in #229 but forgot. This is just a name change to something clearer.